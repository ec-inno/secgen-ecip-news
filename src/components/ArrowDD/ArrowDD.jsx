import React from 'react';

// Project-specific element used as an arrow in a selects.
const ArrowDD = () => (
  <div
    className="ecl-select__icon ecl-u-type-heading-1"
    style={{ color: '#fff' }}
  >
    â–¾
  </div>
);

export default ArrowDD;
