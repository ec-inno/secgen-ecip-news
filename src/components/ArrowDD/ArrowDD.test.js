import React from 'react';
import renderer from 'react-test-renderer';
import ArrowDD from './ArrowDD';

describe('ArrowDD', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ArrowDD />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
