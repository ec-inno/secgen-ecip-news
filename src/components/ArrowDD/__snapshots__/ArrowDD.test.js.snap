// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowDD renders correctly 1`] = `
<div
  className="ecl-select__icon ecl-u-type-heading-1"
  style={
    Object {
      "color": "#fff",
    }
  }
>
  â–¾
</div>
`;
